.github/workflows/build.yaml

```diff
@@ -53,30 +53,21 @@ jobs:
           cmake -B .build -G Ninja \
             -DCMAKE_CXX_COMPILER=${{ matrix.compiler }} \
             -DCMAKE_BUILD_TYPE=Release \
-            -DBUILD_TESTING=ON \
+            -DGRPCXX_BUILD_TESTING=OFF \
```

Comment:
I strongly recommend leaving this to "BULD_TESTING" which is a [standard CMake variable](https://cmake.org/cmake/help/latest/command/enable_testing.html). The reason is that there are many automated package managers (such as buildroot, debian dh_helpers, etc.) that expect to be able to set `BUILD_TESTING` for all packages and for this to result in tests not being built. Also, the expectation is that it propagates also to projects added via `FetchContent`

Reply:
Changed to set `GRPCXX_BUILD_TESTING` when `BUILD_TESTING` is set.